scala/scala-impl/resources/inspectionDescriptions/ConvertibleToMethodValue.html

```diff
@@ -1,24 +1,17 @@
 <html>
 <body>
-Checks if anonymous function can be converted to eta expansion with or without placeholder.
-Also checks if placeholder can be removed from eta expansion.
-
-<p>
-Before:
-<pre>
-def f(x: Any, y: Int) {}
+Reports if anonymous function can be converted to eta expansion with or without placeholder, also checks if placeholder can be removed from eta expansion.
```

Comment:
Reports anonymous functions that can be eta expanded and also reports placeholders that can be removed from eta expansions.